crates/fuel-core/src/graphql_api/database.rs

```diff
@@ -144,6 +151,20 @@ impl ReadView {
         }
     }
 
+    pub async fn transactions(
+        &self,
+        tx_ids: Vec<TxId>,
+    ) -> Vec<StorageResult<Transaction>> {
+        // TODO: Use multiget when it's implemented.
+        let result = tx_ids
+            .iter()
+            .map(|tx_id| self.transaction(tx_id))
+            .collect::<Vec<_>>();
+        // Give a chance to other tasks to run.
+        tokio::task::yield_now().await;
```

Comment:
Nit: I don't think this comment is needed.

```suggestion
        tokio::task::yield_now().await;
```

Reply:
I think comments are useful if you are not familiar with `tokio::task::yield_now()`. I would prefer to keep them

Reply:
Sure, then let's keep them. My thinking is that this is the only purpose of the `yeild_now()` function, so if anyone is unfamiliar with it they'd look it up and then understand that this is the intention. But the comment speeds that process up, so it can be helpful in that sense.